test/functional/services/combo_box.ts

```diff
@@ -16,28 +16,43 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { FtrProviderContext } from '../ftr_provider_context';
+import { WebElementWrapper } from './lib/web_element_wrapper';
 
-export function ComboBoxProvider({ getService, getPageObjects }) {
+export function ComboBoxProvider({ getService, getPageObjects }: FtrProviderContext) {
   const config = getService('config');
   const testSubjects = getService('testSubjects');
   const find = getService('find');
   const log = getService('log');
   const retry = getService('retry');
   const browser = getService('browser');
-  const PageObjects = getPageObjects(['common']);
+  const { common } = getPageObjects(['common']);
```

Comment:
It's pretty common to search for `PageObjects.common` to find usages of this page object, would you mind avoiding destructing so we can continue to rely on that?

Reply:
no problem